xbmc/rendering/dx/DeviceResources.cpp

```diff
@@ -607,7 +607,11 @@ void DX::DeviceResources::ResizeBuffers()
   {
     HDR_STATUS hdrStatus = CWIN32Util::GetWindowsHDRStatus();
     const bool isHdrEnabled = (hdrStatus == HDR_STATUS::HDR_ON);
-    bool use10bit = (hdrStatus != HDR_STATUS::HDR_UNSUPPORTED);
+
+    DXGI_ADAPTER_DESC ad = {};
+    GetAdapterDesc(&ad);
+    bool use10bit = hdrStatus != HDR_STATUS::HDR_UNSUPPORTED &&
```

Comment:
as I understand the sentence it disables 10bit output completely for AMD cards even if HDR is supported. I'm not sure what HDR will work correctly with 8bit output.

Reply:
No, `use10bit` only determines if is used 10 bit for SDR.

For HDR is used `isHdrEnabled`:
https://github.com/xbmc/xbmc/blob/768d7fceac2328fda6e57739e8d598c173f9e503/xbmc/rendering/dx/DeviceResources.cpp#L609

And finally combined condition here:
https://github.com/xbmc/xbmc/blob/768d7fceac2328fda6e57739e8d598c173f9e503/xbmc/rendering/dx/DeviceResources.cpp#L646-L650

Reply:
ok. but what for there is condition with FL >= 12.1? 10bit can be used on FL >= 11

Reply:
Those are just the conditions that trigger it ON by default. They can be overridden by GUI settings in any case. 

The idea is to activate it by default only when used the latest drivers (12.1) from Intel or Nvidia  and when the system also supports HDR (even if it is not activated), this also implies that is in Windows 10/11 only. Systems that do not meet all these conditions may have issues and must be expressly activated by the user (once confirmed that it works).

Instead FL >= 11 is the condition that allows it to be activated. If it fails despite the user forcing it in GUI settings or "try10bitoutput" (Matrix) it will not be enabled.